core/crypto/vrf/vrf_hash_encoder.hpp

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CPP_FILECOIN_CORE_CRYPTO_VRF_VRF_HASH_ENCODER_HPP
+#define CPP_FILECOIN_CORE_CRYPTO_VRF_VRF_HASH_ENCODER_HPP
+
+#include "crypto/vrf/vrf_types.hpp"
+
+namespace fc::crypto::vrf {
+
+  /**
+   * @brief encodeVrfParams function used to create VRFHash according to lotus
```

Comment:
```suggestion
   * @brief encodeVrfParams function is used to create VRFHash according to lotus
```
Also, for me it's not quite clear to me why it's 'encode', but perhaps its okay.

Reply:
because first it encodes parameters in the way specific for lotus implementation and then calculates hash value.